galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/RunImpl.java

```diff
@@ -89,11 +90,19 @@ public RunImpl(String name, IDynamicStatusStoreService dss) throws DynamicStatus
 
         if (test != null) {
             String[] split = test.split("/");
-            this.bundleName = split[0];
-            this.testName = split[1];
+            if(split.length == 2) {
```

Comment:
Can't do this.   Because the gherkin test may be in a folder.
You will have to differentiate between a Java test and a Gherkin test rather that a "test", in a different way.